etc/kayobe/inventory/group_vars/all/firewall

```diff
@@ -0,0 +1,437 @@
+    enabled: "{{ 'vxlan' in (kolla_neutron_ml2_type_drivers + kolla_neutron_ml2_tenant_network_types) and 'network' in group_names }}"
+  # Octavia
+  - rules:
+      - port: 5555/udp
+        zone: "{{ octavia_net_name | net_zone }}"
+        network: "{{ octavia_net_name }}"
+        state: enabled
+    enabled: "{{ kolla_enable_octavia | bool }}"
+  # Overcloud Ironic
+  - rules:
+      - port: 8089/tcp
+        zone: "{{ provision_wl_net_name | net_zone }}"
+        network: "{{ provision_wl_net_name }}"
+        state: enabled
+    enabled: "{{ kolla_enable_octavia | bool }}"
+
+stackhpc_controller_firewalld_rules_extra: []
+
+###############################################################################
+# Compute firewalld rules
+
+stackhpc_compute_firewalld_rules: >-
+  {{ stackhpc_compute_firewalld_rules_default |
+    selectattr('enabled', 'true') |
+    map(attribute='rules') |
+    flatten |
+    selectattr('network', 'in', network_interfaces) |
+    selectattr('zone') |
+    union(stackhpc_compute_firewalld_rules_extra) |
+    unique |
+    select }}
+
+stackhpc_compute_firewalld_rules_default:
+  # Common
+  - rules:
+      - service: ssh
+        zone: "{{ admin_oc_net_name | net_zone }}"
+        network: "{{ admin_oc_net_name }}"
+        state: enabled
+      - service: dhcpv6-client
+        zone: "{{ public_net_name | net_zone }}"
+        network: "{{ public_net_name }}"
+        state: disabled
+    enabled: true
+  - rules:
+      - service: ssh
+        state: disabled
+        zone: "{{ public_net_name | net_zone }}"
+        network: "{{ public_net_name }}"
+    enabled: "{{ public_net_name | net_zone != provision_oc_net_name | net_zone }}"
+  # GENEVE
+  - rules:
+      - port: 6081/udp
+        state: enabled
+        zone: "{{ tunnel_net_name | net_zone }}"
+        network: "{{ tunnel_net_name }}"
+    enabled: "{{ ('geneve' in (kolla_neutron_ml2_type_drivers + kolla_neutron_ml2_tenant_network_types)) | bool }}"
+  # VXLAN
+  - rules:
+      - port: 4789/udp
+        state: enabled
+        zone: "{{ tunnel_net_name | net_zone }}"
+        network: "{{ tunnel_net_name }}"
+    enabled: "{{ ('vxlan' in (kolla_neutron_ml2_type_drivers + kolla_neutron_ml2_tenant_network_types)) | bool }}"
+
+stackhpc_compute_firewalld_rules_extra: []
+
+###############################################################################
+# Storage firewalld rules
+
+stackhpc_storage_firewalld_rules: >-
+  {{ stackhpc_storage_firewalld_rules_default |
+    selectattr('enabled', 'true') |
+    map(attribute='rules') |
+    flatten |
+    selectattr('network', 'in', network_interfaces) |
+    selectattr('zone') |
+    union(stackhpc_storage_firewalld_rules_extra) |
+    unique |
+    select }}
+
+stackhpc_storage_firewalld_rules_default:
+  # Common
+  - rules:
+      - service: ssh
+        zone: "{{ admin_oc_net_name | net_zone }}"
+        network: "{{ admin_oc_net_name }}"
+        state: enabled
+      - port: 9283/tcp
```

Comment:
This one deserves a comment. It's the Ceph prometheus exporter.

Reply:
Maybe also make it conditional on being in the mgrs group and having prometheus enabled.